backend/database/models/officer.py

```diff
@@ -16,16 +16,30 @@ class Rank(str, enum.Enum):
     CHIEF = "CHIEF"
 
 
+class StateID(db.Model):
+    """
+    Represents a Statewide ID that follows an offcier even as they move between
+    law enforcement agencies. for an officer. For example, in New York, this
+    would be the Tax ID Number.
+    """
+    id = db.Column(db.Integer, primary_key=True)
+    officer_id = db.Column(
+        db.Integer, db.ForeignKey("officer.id"))
+    id_name = db.Column(db.Text)  # e.g. "Tax ID Number"
+    state = db.Column(db.Text)  # e.g. "NY"
```

Comment:
Does this make more sense as an enum so we don't have conflicting states like NY and New York?

Reply:
Yes, great point

Reply:
Updated.